js/components/bookmarksToolbar.js

```diff
@@ -152,6 +152,9 @@ class BookmarkToolbarButton extends ImmutableComponent {
       })}
       draggable
       ref={(node) => { this.bookmarkNode = node }}
+      {
+        !this.isFolder ? title={this.props.bookmark.get('location')}
+      }
```

Comment:
This isn't valid syntax and will cause the browser to not start. 
I think you want something like:
`title={this.isFolder ? '' : this.props.bookmark.get('location')}`

it would be good to also have the bookmark title there though too like chrome.